dataset/src/main/scala/frameless/TypedColumn.scala

```diff
@@ -199,7 +209,7 @@ sealed class TypedColumn[T, U](
     *
     * apache/spark
     */
-  def *(u: TypedColumn[T, U])(implicit n: CatalystNumeric[U]): TypedColumn[T, U] = multiply(u)
+  def *(u: TypedColumn[T, U])(implicit n: CatalystNumeric[U], tt: ClassTag[U]): TypedColumn[T, U] = multiply(u)
```

Comment:
@OlivierBlanvillain, is this tt ClassTag needed?

Reply:
It's used here: https://github.com/typelevel/frameless/pull/180/files#diff-a11540c4ef162be102a1fb83f423c811R194